src/User.js

```diff
@@ -18,4 +18,4 @@ class User {
 
 if (typeof module !== 'undefined') {
   module.exports = User;
-}
+};
```

Comment:
love a good semicolon!

Reply:
I need to make sure that I'm adding them as I go along. Thank you for the reminders!